### ISSUE TITLE
Standard Deviation Schedule

### ISSUE THREAD

**Issue by rschlos-snl** [DLR-RM/stable-baselines3#231]
### Question

Is it possible to devise a standard deviation schedule, similar to lr_schedule, using stable-baselines3?

### Additional context

I have found it beneficial to train my agent and save the model, reload the model, reset the standard deviation, and continue training:

`model = PPO.load(log_dir, env=env, policy_kwargs=policy_kwargs)`
`model.policy.log_std=th.nn.Parameter(th.tensor([-0.9], device='cuda:0', requires_grad=True))`
`model.save("ppo_3dof_temp")`
`model = PPO.load("ppo_3dof_temp", env=env, policy_kwargs=policy_kwargs)`
`model.learn(total_timesteps=int(timesteps), callback=callback, tb_log_name='ppo')`

I would like to know if there is a way to implement this without having to stop and re-start training.

### Checklist

- [ x] I have read the [documentation](https://stable-baselines3.readthedocs.io/en/master/) (**required**)
- [ x] I have checked that there is no similar [issue](https://github.com/DLR-RM/stable-baselines3/issues) in the repo (**required**)

**Comment by araffin** [comment]
Hello,

You should probably consider using a [Callback](https://stable-baselines3.readthedocs.io/en/master/guide/callbacks.html)
and set `requires_grad=False` if you don't want the standard deviation to be updated by the policy loss.

Related issue: https://github.com/DLR-RM/stable-baselines3/issues/155
